container.go

```diff
@@ -167,6 +167,9 @@ func (c *Container) AwaitListening() bool {
 		url := fmt.Sprint("http://", port.IP, ":", port.PublicPort, "/")
 		for {
 			response, err := http.Get(url)
+			if response != nil && response.Body != nil {
+				response.Body.Close()
```

Comment:
Oh wow! You really have to read the docs carefully to use Go :(

Am I right that the only way of knowing this would be to have properly read right through to the last senhttp://golang.org/pkg/net/http/#Client.Get where it says "Caller should close resp.Body when done reading from it"?